runtimes/configrt/runtime/run.go

```diff
@@ -28,29 +28,29 @@ func (r *run) getFunc(ctx context.Context, args []sdktypes.Value, kwargs map[str
 	var key string
 
 	if err := sdkmodule.UnpackArgs(args, kwargs, "key", &key); err != nil {
-		return nil, err
+		return sdktypes.InvalidValue, err
 	}
 
 	v, ok := r.exports[key]
 	if !ok {
-		return sdktypes.NewNothingValue(), nil
+		return sdktypes.Nothing, nil
 	}
 
 	return v, nil
 }
 
 func (r *run) Call(ctx context.Context, v sdktypes.Value, args []sdktypes.Value, kwargs map[string]sdktypes.Value) (sdktypes.Value, error) {
```

Comment:
could we return pointer instead of type and then (i) follow go convention to return `nil, err` in case of error and (ii) prevent by value objects copy?

Reply:
The objects are just wrappers around a pointer. See what I wrote in https://github.com/autokitteh/autokitteh/pull/44#discussion_r1511960443. I guess we'll have to discuss it in sync.